src/adapters/supabase/helpers/client.ts

```diff
@@ -340,6 +347,20 @@ export const getAccessLevel = async (username: string, repository: string, label
   return accessValues;
 };
 
+export const getAllAccessLevels = async (username: string, repository: string): Promise<false | accessLevels> => {
+  const logger = getLogger();
+  const { supabase } = getAdapters();
+
+  const { data } = await supabase.from("access").select("*").eq("user_name", username).eq("repository", repository).single();
+
+  if (!data) {
+    logger.info(`Access not found on the database`);
+    // no access
+    return false;
```

Comment:
```suggestion
    return null;
```
I think `null` is more appropriate

Reply:
it can be either `null` or `undefined`. Literally it doesn't make sense to return `boolean` type